ui/desktop/app/controllers/scopes/scope/projects/targets/index.js

```diff
@@ -60,7 +60,13 @@ export default class ScopesScopeProjectsTargetsIndexController extends Controlle
 
     // Associate the connection details with the session
     let session;
-    const { session_id, address, port, credentials } = connectionDetails;
+    const {
+      session_id,
+      address,
+      port,
+      credentials,
+      expiration_time: expiration,
```

Comment:
```suggestion
      expiration: expiration_time,
```

I think you have it mixed up